modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -90,6 +91,12 @@ static struct
 
     /* Icon group file containing one PNG icon and one normal icon */
     {L"%temp%\\ROS.ico", 1, 1, TRUE},
+
+    /* Executable file with bad 'Icon Group' but good 'Icons'.
+     * Windows explorer shows the program's icon correctly in WinXP/Win2K3
+     * but Windows 7 shows only a default icon. This is analogous
```

Comment:
Would someone know what Vista shows?

Reply:
I am not aware of any active ros developer operating (or owning) that OS. I think it is acceptable to add that information whenever the need would evolve. We already do know that different versions of Windows do behave differently here, that is what I would consider to be most important for now. After all it is most likely that if we ever aimed to retarget ros in the future, that we will entirely skip the Vista-intermediate-step and jump directly to 7 or even higher.
Unless you could provide the answer yourself, I don't think it makes sense to postpone merging the test just due to that unresolved question.

Reply:
We have testers with Vista VMs, just ask if needed.

Reply:
I just installed Windows Vista and it only shows the default icon.